system/adcscope/Kconfig

```diff
@@ -0,0 +1,105 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig SYSTEM_ADCSCOPE
+	bool "NxScope ADC stream app"
+	default n
+	depends on ADC && LOGGING_NXSCOPE
+
+if SYSTEM_ADCSCOPE
+
+config SYSTEM_ADCSCOPE_PROGNAME
+	string "Program name"
+	default "adcscope"
+
+config SYSTEM_ADCSCOPE_PRIORITY
+	int "adcscope task priority"
+	default 100
+
+config SYSTEM_ADCSCOPE_STACKSIZE
+	int "adcscope stack size"
+	default DEFAULT_TASK_STACKSIZE
+
+config SYSTEM_ADCSCOPE_SERIAL_PATH
+	string "adcscope serial device path"
+	default "/dev/ttyUSB0"
+
+config SYSTEM_ADCSCOPE_SERIAL_BAUD
+	int "adcscope serial baud"
+	default 115200
+	---help---
+		Ignored if set to 0 (for example for RTT interface)
+
+config SYSTEM_ADCSCOPE_CDCACM
+	bool "adcscope CDCACM device support"
+	depends on CDCACM
+	default n
+
+config SYSTEM_ADCSCOPE_FORCE_ENABLE
+	bool "adcscope force enable"
+	default n
+
+config SYSTEM_ADCSCOPE_STREAMBUF_LEN
+	int "adcscope stream buffer length"
+	default 512
+
+config SYSTEM_ADCSCOPE_RXBUF_LEN
+	int "adcscope RX buffer length"
+	default 32
+
+config SYSTEM_ADCSCOPE_RX_PADDING
+	int "adcscope RX padding"
+	default 0
```

Comment:
Please include a help here, since padding it not obvious and/or could depends on "external" factors such as ADC resolution, etc

Reply:
this is internal nxscope parameter that can be used to optimize RX transfers from nxscope client (from host software to adcscope app). It has nothing to do with ADC.

Reply:
Hmm, thank you for explanation. Please include a help to let user know what is it and when/why to use it